### PR TITLE
cloud: update release notes for 914

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
cloud: update release notes for 914

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/release-notes-2022.md [hunk 1]
```diff
@@ -8,6 +8,26 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## September 14, 2022
+
+**General changes**
+
+* Support connecting to TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters via TLS
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:15]:
```suggestion
* Support connecting to TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters via TLS.
```

File: tidb-cloud/release-notes-2022.md [hunk 2]
```diff
@@ -8,6 +8,26 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## September 14, 2022
+
+**General changes**
+
+* Support connecting to TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters via TLS.
+
+    For Dedicated Tier clusters, the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB Cloud Root CA certificate and also provides the connection string and sample code for one-way TLS connection to TiDB clusters. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:17]:
```suggestion
    For Dedicated Tier clusters, the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for one-way TLS connection to TiDB clusters. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
```

Comment (zhuangliangji1992) [reply tidb-cloud/release-notes-2022.md]:
```suggestion
    also provides the connection string and sample code for TLS connection.
```

File: tidb-cloud/release-notes-2022.md [hunk 3]
```diff
@@ -8,6 +8,26 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## September 14, 2022
+
+**General changes**
+
+* Support connecting to TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters via TLS.
+
+    For Dedicated Tier clusters, the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for one-way TLS connection. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:17]:
```suggestion
    For Dedicated Tier clusters, the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for TLS connection. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
```

File: tidb-cloud/release-notes-2022.md [hunk 4]
```diff
@@ -8,6 +8,26 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+**General changes**
+
+* Support connecting to TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters via TLS.
+
+    For Dedicated Tier clusters, the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for TLS connection. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
+
+**Console changes**
+
+* Re-design the web interface of the cluster overview page and the [Clusters](https://tidbcloud.com/console/clusters) page for better user experience.
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:21]:
```suggestion
* Re-design the web interface of the cluster overview page and the [Clusters](https://tidbcloud.com/console/clusters) page for better user experience.

      In the new interface, the entrances of upgrade to Dedicated Tier, cluster connection, and data import are highlighted.

```

File: tidb-cloud/release-notes-2022.md [hunk 5]
```diff
@@ -8,6 +8,29 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    In the new interface, the entrances of upgrade to Dedicated Tier, cluster connection, and data import are highlighted.
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:23]:
```suggestion
* Optimize the UI of the cluster overview page and the [Clusters](https://tidbcloud.com/console/clusters) page, and highlight the entrances of upgrade to Dedicated Tier, cluster connection, and data import are highlighted.
```

File: tidb-cloud/release-notes-2022.md [hunk 6]
```diff
@@ -8,6 +8,27 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+**General changes**
+
+* Support connecting to TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters via TLS.
+
+    For Dedicated Tier clusters, the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for TLS connection. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
+
+**Console changes**
+
+* Optimize the UI of the cluster overview page and the [Clusters](https://tidbcloud.com/console/clusters) page, and highlight the entrances of upgrade to Dedicated Tier, cluster connection, and data import.
+
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:22]:
```suggestion
```

**Review by langyuezhang — APPROVED** [review]
LGTM

File: tidb-cloud/release-notes-2022.md [hunk 7]
```diff
@@ -8,6 +8,26 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+**General changes**
+
+* Support connecting to TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters via TLS.
+
+    For Dedicated Tier clusters, the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for TLS connection. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
+
+**Console changes**
+
+* Optimize the UI of the cluster overview page and the [Clusters](https://tidbcloud.com/console/clusters) page, and highlight the entrances of upgrade to Dedicated Tier, cluster connection, and data import.
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:21]:
```suggestion
* Optimize the UI of the [Clusters](https://tidbcloud.com/console/clusters) page and the cluster overview page for better user experience. In the new design, the entrances of upgrade to Dedicated Tier, cluster connection, and data import are highlighted.
```

File: tidb-cloud/release-notes-2022.md [hunk 8]
```diff
@@ -8,6 +8,26 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+**General changes**
+
+* Support connecting to TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters via TLS.
+
+    For Dedicated Tier clusters, the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for TLS connection. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
+
+**Console changes**
+
+* Optimize the UI of the [Clusters](https://tidbcloud.com/console/clusters) page and the cluster overview page for better user experience. In the new design, the entrances of upgrade to Dedicated Tier, cluster connection, and data import are highlighted.
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:21]:
```suggestion
* Optimize the UI of the [Clusters](https://tidbcloud.com/console/clusters) page and the cluster overview page for better user experience.

    In the new design, the entrances of upgrade to Dedicated Tier, cluster connection, and data import are highlighted.
```

File: tidb-cloud/release-notes-2022.md [hunk 9]
```diff
@@ -8,6 +8,28 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## September 14, 2022
+
+**General changes**
+
+* Support connecting to TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters via TLS.
+
+    For Dedicated Tier clusters, the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for TLS connection. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:17]:
```suggestion
    For Dedicated Tier clusters, the **Standard Connection** tab in [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for TLS connection. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
```

**Review by Frank945946 — APPROVED** [review]
LGTM

File: tidb-cloud/release-notes-2022.md [hunk 10]
```diff
@@ -8,6 +8,28 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+**Console changes**
+
+* Optimize the UI of the [Clusters](https://tidbcloud.com/console/clusters) page and the cluster overview page for better user experience.
+
+    In the new design, the entrances of upgrade to Dedicated Tier, cluster connection, and data import are highlighted.
+
+* Provide a new web UI for data import. The new UI provides better user experience and makes data import more efficient.
+
+    Using the new UI, you can preview the file data to be imported, view the import process, and manage all import tasks easily.
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:27]:
```suggestion
    Using the new UI, you can preview the data to be imported, view the import process, and manage all import tasks easily.
```

File: tidb-cloud/release-notes-2022.md [hunk 11]
```diff
@@ -8,6 +8,28 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## September 14, 2022
+
+**General changes**
+
+* Support connecting to TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters via TLS.
+
+    For Dedicated Tier clusters, the **Standard Connection** tab in [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for TLS connection. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:17]:
```suggestion
    For Dedicated Tier clusters, the **Standard Connection** tab in the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for TLS connection. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
```

File: tidb-cloud/release-notes-2022.md [hunk 12]
```diff
@@ -8,6 +8,28 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## September 14, 2022
+
+**General changes**
+
+* Support connecting to TiDB Cloud [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters via TLS.
+
+    For Dedicated Tier clusters, the **Standard Connection** tab in the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for TLS connection. You can connect to your Dedicated Tier cluster via TLS using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:17]:
```suggestion
    For Dedicated Tier clusters, the **Standard Connection** tab in the [Connect](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) dialog now provides a link to download the TiDB cluster CA and also provides the connection string and sample code for TLS connection. You can [connect to your Dedicated Tier cluster via TLS](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) using third-party MySQL clients, MyCLI, and multiple connection methods for your applications, such as JDBC, Python, Go, and Node.js. This feature ensures the security of data transmission from your applications to TiDB clusters.
```

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/release-notes-2022.md [hunk 13]
```diff
@@ -8,6 +8,18 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## September 14, 2022
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:11]: Is this the existing release note published last week?

**Pull request merged**